src/hearings/resolvers/priority/priority.resolve.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { ActivatedRouteSnapshot, Resolve, Router, RouterStateSnapshot } from '@angular/router';
+import { RefDataModel } from 'api/hearings/models/refData.model';
+import { forkJoin, Observable } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+import { HearingsDataService } from 'src/hearings/services/hearings-data.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class PriorityResolver implements Resolve<{ priorities: RefDataModel[] }> {
+  constructor(
+    private readonly router: Router,
+    private readonly hearingsDataService: HearingsDataService
+  ) { }
+
+  public resolve(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Observable<{ priorities: RefDataModel[] }> {
+    const priorities$ = this.hearingsDataService.getPriorities().pipe(
+      catchError(error => {
+        return [];
+      })
+    ) as Observable<RefDataModel[]>;
+
+    return forkJoin({ priorities: priorities$ });
```

Comment:
Why are you using a folkjoin? A folkjoin should have an array off request/submissions I can only see one.